language-support/ts/codegen/tests/ts/yarn.lock

```diff
@@ -1,4314 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why did you delete this? afaik this is used when running the tests outside of Bazel.

Reply:
sorry not done yet -- switched to draft, still checking a few things

Reply:
hiya @cocreature long discussion with @shayne-fletcher about this today.  conclusion is it is not really needed as that folder has some auto-magic yarn installin' that happens in a temp folder so the lock file provides little benefit, and the yarn utilities don't really work standalone because of the dynamic codegen dependencies that come in

Reply:
so i think we are okay to delete it unless any objections @hurryabit ?

Reply:
No objections.